dist/bin/no-internal-report.js

```diff
@@ -1,64 +1,39 @@
 #! /usr/bin/env node
-
 /*---------------------------------------------------------------------------------------------
 * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
 * See LICENSE.md in the project root for license terms and full copyright notice.
 *--------------------------------------------------------------------------------------------*/
 "use strict"
 const path = require('path');
-const fs = require('fs');
-
-function getNodeModulesDir(startPath) {
-  let currPath = startPath;
-  let baseName = path.basename(currPath);
-  while (baseName !== "node_modules") {
-    currPath = path.resolve(currPath, "..")
-    if (currPath === "/")
-      return undefined;
-    baseName = path.basename(currPath);
-  }
-  return currPath;
-}
-
-const nodeModules = getNodeModulesDir(path.dirname(process.argv[1]));
-if (!nodeModules)
-  throw ("Could not find node_modules directory");
+const assert = require('assert');
+const eslintBinPath = path.join(require.resolve("eslint"), "../../bin/eslint.js");
 
-const distDir = path.join(nodeModules, "@itwin/eslint-plugin/dist")
-if (!fs.existsSync(distDir))
-  throw ("Could not find required dir: " + distDir);
-
-const configDir = path.join(nodeModules, "@itwin/eslint-plugin/dist/bin/eslint.config.js")
-if (!fs.existsSync(distDir))
-  throw ("Could not find required dir: " + configDir);
-
-let tags;
-if (process.argv.length > 3) {
-  tags = process.argv[2].toString();
+const argCount = process.argv.length;
+try {
+  assert(argCount >= 3 && argCount <= 4);
```

Comment:
??

Reply:
The purpose here is to make sure that we either get something that has no tags supplied which would have 3 args, or 4 when tags are supplied to it

Reply:
then why not `argCount === 3 || argCount === 4` ?

Reply:
Rather, let's allow any amount of positional arguments like eslint does. Instead, check for a `--tags` option to find which elements of process.argv should not be passed to eslint

Reply:
changed to this setup. Added a check for a --files arg

Reply:
requiring the `--files` option scales poorly.
You'd have to type it again and again for each additional path:
```sh
no-internal-report --tags beta,alpha --files 'frontend/**' --files 'backend/**' --files 'common/**'
```
It is perfectly normal to have "positional arguments" (arguments not following a `--options`) separate from optional options that use `--long-form` or `-l` (short form) indicators.

In my example above, if it is not preceded by the `--tags` option indicator, it is a positional argument. I think that scales better and recommend following that instead.

Reply:
the current iteration throws an error when specifying more than one path glob. Looking into a way to allow more than one. Will make the move back to positional args

Reply:
changed to have a check for only the --tags arg, rest are loaded into the files array like in your example. Works when supplying multiple file globs